sizefashion/src/components/ProductImages.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { getImageTags } from '../utils/helpers';
+
+const ProductImages: React.FC<{ image: string | undefined }> = ({ image = '' }) => {
```

Comment:
@otaaviostein if you're rendering an `img` and not a **set** of `img`, maybe we can rename this component to `ProductImage` instead?